### DISCUSSION TITLE
Show time when car was parked

### DISCUSSION THREAD

**Discussion by garo** [teslamate-org/teslamate#3149] [Ideas]
It would be great if the teslamate web page would display the time when the car was last parked.

In Finland it's common to have parking spaces which are free eg. for one, two or four hours. It would be handy to check when the car was parked with a quick glance without having to remember by myself, or by going to Grafana and to look the most recent trip.

**Comment by garo** [comment]
I think I might be able to do this by myself, but I've never written any Elixir, so I would need some pointers which files I should touch etc?

**Comment by DrMichael** [comment]
You could see it in the Timeline dashboard...

**Reply by garo** [reply]
Thanks! I actually did not realise this. With the Timeline dashboard checking the parking time is behind three additional clicks, which I think is good enough.
